### PR TITLE
Marketplace scan report

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
@KseniiaIankina please take a look and leave comments if necessary

File: content/appstore/general/share-app-store-content.md [hunk 1]
```diff
@@ -170,6 +170,16 @@ If this is the first version of the component you are uploading, the number in t
 
 Enter **Release Notes** for the component in the box provided describing what is new in that release.
 
+Finally, you can upload a **Virus Scan Report**. Uploading a scan report is currently optional, but Mendix strongly recommends that you create this report using [VirusTotal](https://www.virustotal.com/gui/home/upload), for example. This will ensure a high-quality and secure component. Once you upload your *.mpk* package or other component file and run the scan, save the report as a PDF. Click **Scan Report Upload** to upload your report.
+
+{{% alert type="warning" %}}
+If the virus scan report is invalid or there is some issue with it, the [Marketplace approval proecess](/appstore/general/share-app-store-content#approval) will reject the component and it will not be listed in the Marketplace. Ensure you upload a true virus scan report to avoid the rejection of your component.
```
Comment (KseniiaIankina) [inline content/appstore/general/share-app-store-content.md]: Link - there is a typo "Marketplace approval process"

File: content/appstore/general/share-app-store-content.md [hunk 2]
```diff
@@ -170,6 +170,16 @@ If this is the first version of the component you are uploading, the number in t
...
+{{% /alert %}}
+
+{{% alert type="warning" %}}
+If this report is not attached and during our approval process we identify security issues in your component, the component will not be listed in the public Marketplace until the issue is resolved.
+{{% /alert %}}
+
 ![](attachments/share-app-store-content/package.jpg)
```
Comment (KseniiaIankina) [inline content/appstore/general/share-app-store-content.md:183]: Rajesh will add a different attachment - latest one

**Review by KseniiaIankina — CHANGES_REQUESTED** [review]
Thank you Adam for preparing this documentation piece ! Could you please correct one typo and then it is approved.

File: content/appstore/general/share-app-store-content.md (hunk 2 above)
Comment (Adam-Dupaski) [reply content/appstore/general/share-app-store-content.md:183]: Okay, please ensure it uses the same example data.

**Comment by Adam-Dupaski** [comment]
Merged #4225

**Pull request closed without merging**